spec/rspec/support/object_formatter_spec.rb

```diff
@@ -158,6 +158,13 @@ def with_delegate_loaded
             matcher_without_a_description.inspect)
         end
       end
+
+      context 'with truncation' do
+        it 'produces an output of limited length' do
+          formatter = ObjectFormatter.new(10)
```

Comment:
At the moment, nil disables it. Is -1 required? If so, should I remove the nil behaviour? Should it be -1 explicitly or any negative number?